### DISCUSSION TITLE
Wiki needs updating

### DISCUSSION THREAD

**Discussion by tkocou** [Guilouz/Klipper-Flsun-Speeder-Pad#139] [General]
I am following the wiki page [https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/wiki/Install-Official-Builds-(1-instance)](url) And it mentions "...  go to your Mainsail Web interface then select the Machine tab. Open the moonraker.conf file and add the following lines at the end of the file..."
How does one get access to posted interface? The speeder-pad does not show any sort of menu as described. Checking by SSH, there is no clear directory indicated as the official "Web interface" directory. The further instructions seem to indicate that one would do the editing via SSH. Any wisdom about how to accomplish this portion of the wiki page?

**Comment by tkocou** [comment]
I believe I have figured out what piece of information is missing from the wiki page.
I suggest change this: _After reboot, go to your Mainsail Web interface then select the Machine tab._
To this: _**After reboot, using a web browser, go to your Mainsail Web interface then select the Machine tab.**_

**Reply by Guilouz** [reply]
Mainsail is a web interface so it makes sense to use a browser for this…
